runtime/local/src/chain_extensions.rs

```diff
@@ -28,9 +28,10 @@ pub use pallet_chain_extension_xvm::XvmExtension;
 
 // Following impls defines chain extension IDs.
 
-impl RegisteredChainExtension<Runtime> for DappsStakingExtension<Runtime> {
-    const ID: u16 = 00;
-}
+// This can be restored when integrated with dApp staking v3.
```

Comment:
I think you can remove everything related to dapp staking chain extension (in local in shibuya). We can still update it if there is a need for it

Reply:
Sure, will do that.